integration_tests/__tests__/version.test.js

```diff
@@ -4,13 +4,15 @@
  * This source code is licensed under the BSD-style license found in the
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
```

Comment:
that line isn't supposed to be here, i'll remove it :)
I'm trying to type tests in some projects at fb, but that will require a lot of work on on jest side.
I'll submit a diff with some flowtypes for jest tests, but i'm not sure what's @cpojer's vision on this is